src/crimson/osd/osd.cc

```diff
@@ -575,7 +575,7 @@ seastar::future<> OSD::committed_osd_maps(version_t first,
       logger().info("osd.{}: now preboot", whoami);
 
       if (m->get_source().is_mon()) {
-        return _preboot(m->oldest_map, m->newest_map);
+        return _preboot(m->newest_map, m->oldest_map);
```

Comment:
@liu-chunmei thanks! i think it would be better to transpose these two parameters on the callee side instead. that'd be more intuitive and less error prone -- oldest first, and newest later.

what do you think?

Reply:
That is what I think. will update it.